clang/docs/ClangFormatStyleOptions.rst

```diff
@@ -3311,22 +3311,22 @@ the configuration (without a prefix: ``Auto``).
 
 .. _IndentGotoLabels:
 
-**IndentGotoLabels** (``Boolean``) :versionbadge:`clang-format 10` :ref:`¶ <IndentGotoLabels>`
+**IndentGotoLabels** (``enum``) :versionbadge:`clang-format 10` :ref:`¶ <IndentGotoLabels>`
   Indent goto labels.
 
-  When ``false``, goto labels are flushed left.
+  When ``GLI_None``, goto labels are flushed left.
```

Comment:
Do you edit this by hand?
`ClangFormatStyleOptions.rst` is generated by clang/docs/tools/dump_format_style.py.

So do all your edits in `Format.h`, and then run the script.

Reply:
1> yes. I did not know that.
2> okay. Got it.

Reply:
I will do it correctly in the morning.